control/vehicle_cmd_gate/src/vehicle_cmd_gate.cpp

```diff
@@ -229,25 +229,29 @@ void VehicleCmdGate::onAutoTurnIndicatorsCmd(
   autoware_auto_vehicle_msgs::msg::TurnIndicatorsCommand::ConstSharedPtr msg)
 {
   auto_commands_.turn_indicator = *msg;
+  auto_commands_.turn_indicator.stamp = this->now();
```

Comment:
I think the publisher should set the timestamp information in this message. :thinking:

Reply:
Instead,  should I use `this->get_clock()->now()`?

Reply:
Mm, no. Instead of overwriting the timestamp here, I believe `autoware_auto_vehicle_msgs::msg::TurnIndicatorsCommand::ConstSharedPtr msg` should have the timestamp.

Reply:
@TakaHoribe has the same comment.
https://github.com/autowarefoundation/autoware.universe/pull/747#issuecomment-1105064737